DATA/MISSIONS/mBases.ini

```diff
@@ -41487,3 +41487,1516 @@ character_density = 3
 fixture = start06_fix_ship, Zs/NPC/Shipdealer/01/A/Stand, scripts\vendors\li_shipdealer_fidget.thn, ShipDealer
 fixture = start06_fix_trader, Zs/NPC/Trader/01/A/Stand, scripts\vendors\li_commtrader_fidget.thn, trader
 fixture = start06_fix_weaponsdealer, Zs/NPC/Equipment/01/A/Stand, scripts\vendors\li_equipdealer_fidget.thn, Equipment
+
+
+;; Proxy Base
+;; Carrier
+;; Rheinland 
+
+[MBase]
+nickname = Br_Proxy_Rh_Battleship_Base
+local_faction = rh_n_grp

```

Comment:
The factions of the base and NPCs should be all neutral. E.g. fc_uk_grp or Freelancer rep. Because we are not really on a base run by a faction, but by a player.